singletons/utils.js

```diff
@@ -156,19 +156,75 @@ module.exports = (function (Module) {
 		 * Returns a formatted string, specifying an amount of time delta from current date to provided date.
 		 * @param {sb.Date|Date|number} target
 		 * @param {boolean} [skipAffixes] if true, the affixes "in X hours" or "X hours ago" will be omitted
+		 * @param {sb.Date} [deltaTo] If set, calculate time delta between target and deltaTo. If undefined, calculate
+		 * delta between target and the current time.
 		 * @returns {string}
 		 */
-		timeDelta (target, skipAffixes = false) {
-			const now = new sb.Date();
-			if (sb.Date.equals(now, target)) {
+		timeDelta (target, skipAffixes = false, deltaTo = undefined) {
+			if (deltaTo === undefined) {
+				deltaTo = new sb.Date();
+			}
+
+			if (sb.Date.equals(deltaTo, target)) {
 				return "right now!";
 			}
 
+			if (typeof target === "number") {
```

Comment:
Just a small tidbit - Number.prototype.valueOf is valid as well, so there's no need to differentiate this. Unless I'm missing something?